src/lib/types.ts

```diff
@@ -71,8 +71,16 @@ export interface PollImportResponse {
 }
 
 // used to generate import data by connecting to the source via API
-// and listing all orgs + repos / targets
-export enum SupportedIntegrationTypesToGenerateImportData {
+// and listing all repos / targets per given org
+export enum SupportedIntegrationTypesImportData {
```

Comment:
separate the types so the support for listing repos & orgs can be introduced separately